x-pack/test/functional/page_objects/security_page.js

```diff
@@ -31,10 +31,6 @@ export function SecurityPageProvider({ getService, getPageObjects }) {
 
       await PageObjects.common.navigateToApp('login');
 
-      // ensure welcome screen won't be shown. This is relevant for environments which don't allow
```

Comment:
Did you test whether it's safe to remove this from here? I'm not sure whether all tests go through the route above, but it might be the case.

Reply:
I'll try to give it a full run on a cloud production cluster, but there are known test failures on cloud (https://github.com/elastic/kibana/projects/20), so I'm not sure if I can get a clean result.

Reply:
The `security_page` has been changed recently in #65705 and the welcome screen disabling has been added a second time to it. In order to play it on the safe side, I won't touch that bit for now. I've rolled back my `security_page` changes in a38a39d and adjusted issue title + description accordingly. I've also removed the environment variable usage in 7605062 as all other places in functional tests (`security_page`, yaml file) would not take that into account and it doesn't seem to make sense to just have an env variable for one of the pathes.